test/unit/region.spec.js

```diff
@@ -901,18 +901,15 @@ describe('region', function() {
         })
       });
 
-      this.MyView = Backbone.View.extend({
-        render: function() {
-          $(this.el).html('some content');
-        },
-
-        destroy: function() {}
+      this.MyView = Backbone.Marionette.View.extend({
```

Comment:
Tests are wip. We should test BB.view as before, but should  also test MN.view were we are not calling remove